sort.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+function bubbleSort($sort_array)
+{
+    $array_indexes = count($sort_array);
+    $lastIndex =  $array_indexes - 1;
+    if (strlen($sort_array[$lastIndex]) < 2) {
```

Comment:
For the sake of readability please separate these two lines ("$lastIndex..." and "if...") by an empty line.